backend/census_historical_migration/workbooklib/notes_to_sefa.py

```diff
@@ -4,7 +4,6 @@
     set_single_cell_range,
     map_simple_columns,
     generate_dissemination_test_table,
-    test_pfix,
```

Comment:
Does this not have to be replaced with str, rather than merely removing test_pfix?

Reply:
Nope, `str` is a built-in Python [method](https://www.w3schools.com/python/ref_func_str.asp)